src/pfs/pfs.proto

```diff
@@ -252,23 +241,26 @@ message CommitInfos {
   repeated CommitInfo commit_info = 1;
 }
 
-message SquashCommitRequest {
-  Commit commit = 1;
+message InspectJobRequest {
+  Job job = 1;
 }
 
-message FlushCommitRequest {
-  repeated Commit commits = 1;
-  repeated Repo to_repos = 2;
+message SquashJobRequest {
+  Job job = 1;
+}
+
+message FlushJobRequest {
```

Comment:
We discussed rolling `FlushJob`s semantics in `InspectJob` on the call. Just recording that here I think we were all on board with it.